plugins/net_plugin/net_plugin.cpp

```diff
@@ -2924,10 +2924,7 @@ namespace eosio {
    }
 
    size_t calc_trx_size( const packed_transaction_ptr& trx ) {
-      // transaction is stored packed and unpacked, double packed_size and size of signed as an approximation of use
-      return (trx->get_packed_transaction().size() * 2 + sizeof(trx->get_signed_transaction())) * 2 +
-             trx->get_packed_context_free_data().size() * 4 +
-             trx->get_signatures().size() * sizeof(signature_type);
+      return trx->get_estimated_size();
```

Comment:
- packed_trx is only transaction (does not include signed_transaction parts)
- a packed_transaction has a packed_trx and a signed_transaction as part of it
- So *2 for signatures and context_free_data  because they are in unpacked_trx

Update the comment to hopefully make it clearer.